src/serializer/functions.js

```diff
@@ -32,11 +33,14 @@ import { ReactStatistics } from "./types";
 import type { AdditionalFunctionEffects, WriteEffects } from "./types";
 import { convertConfigObjectToReactComponentTreeConfig, valueIsKnownReactAbstraction } from "../react/utils.js";
 import { applyOptimizedReactComponents, optimizeReactComponentTreeRoot } from "../react/optimizing.js";
+import { ShapeInformation } from "../utils/ShapeInformation";
+import type { ArgModel } from "../utils/ShapeInformation";
 import * as t from "babel-types";
 
 type AdditionalFunctionEntry = {
   value: ECMAScriptSourceFunctionValue | AbstractValue,
   config?: ReactComponentTreeConfig,
+  argModelString?: StringValue,
```

Comment:
I don't think there's a need to keep around the `StringValue` wrapper. Just store the contained `string`.